cli/src/print.rs

```diff
@@ -242,5 +251,39 @@ mod tests {
                 ],
             }
         );
+
+        test!(
+            "
+╭──────────────╮
+│ Field   Type │
+╞══════════════╡
+│ id      Int  │
+│ name    Text │
+╰──────────────╯",
+            Payload::ShowColumns {
+                rows: vec![
+                    vec!["id".to_string(), "Int".to_string(),],
+                    vec!["name".to_string(), "Text".to_string(),],
+                ],
+            }
+        );
+
+        test!(
+            "
+╭────────────────╮
+│ Field     Type │
+╞════════════════╡
+│ id        Int  │
+│ name      Text │
+│ isabear   Bool │
+╰────────────────╯",
+            Payload::ShowColumns {
+                rows: vec![
+                    vec!["id".to_string(), "Int".to_string(),],
+                    vec!["name".to_string(), "Text".to_string(),],
+                    vec!["isabear".to_string(), "Bool".to_string(),],
+                ],
+            }
```

Comment:
These two test cases look redundant.
Lack of tests are the issue, but we also don't need to add redundant ones.
Let's use the last one which has more info.